src/Type/Row.purs

```diff
@@ -0,0 +1,27 @@
+module Type.Row
+  ( class RowLacks
+  ) where
+
+-- Must not be exported
+foreign import data RowLacking :: Symbol -> Symbol -> Type
+
+class RowLacksF (key :: Symbol)
```

Comment:
Ah yes, if I change `RowLacksF` so that it expects `rest = row` then it does catch that case. But it still passes for `forall t. RowLacks "k" (k :: t) => Int`.